betterproto/tests/test_inputs.py

```diff
@@ -75,11 +84,19 @@ def test_data(request):
 
     sys.path.append(reference_module_root)
 
+    plugin_module = importlib.import_module(f"{plugin_output_package}.{test_case_name}")
+
+    plugin_module_entry_point = find_module(plugin_module, module_has_entry_point)
+
+    if not plugin_module_entry_point:
+        raise Exception(
+            f"Test case {repr(test_case_name)} has no entry point. "
+            + "Please add a proto message or service called Test and recompile."
```

Comment:
don't need the + here

Reply:
implicit concatenation? didn't know that! thanks!